src/app/components/details/details.rs

```diff
@@ -193,26 +150,47 @@ impl Details {
 
         let modal = ReleaseDetailsWindow::new();
 
-        widget.connect_liked(clone!(@weak model => move || model.toggle_save_album()));
+        widget
+            .header_widget()
+            .connect_liked(clone!(@weak model => move || model.toggle_save_album()));
```

Comment:
might be worth adding a connect_liked fn to the details widget

Reply:
(I can definitely help you with these not so fun tasks if needed)

Reply:
Yeah, that would be great if you don't mind.